README.md

```diff
@@ -76,33 +76,42 @@ $ adr-viewer --help
 
 Using adr-tools cli (adr):
 
-### set adr-tools directory for team in question:
-See the limitations of adr-tools notation above. You need to change `.adr-dir` to the team you want to work on.
-```
-$ echo "decisions/websre" > .adr-dir
-```
+To handle multiple teams we've made the table of contents generator work just on filenames.
+As such you can generally treat each adr directory as independent from the point of view of the ADR CLI tool.
 
 ### create new adr proposal for your team:
+This assumes someone has initialized a team or group folder for you previously.
+If not, the info is below.
+
 ```
+$ cd decisions/$team
 $ adr new My Awesome Web SRE Team ADR
   ./0002-my-awesome-global-adr.md
 # then add your info & go
 $ vi 0002-my-awesome-global-adr.md
 ```
 
 ### initialize a new team- or group-specific decisions directory:
+
+You'll need a new folder for your team, feel free to add as needed.
+You will need to force add the adr-dir, as we want to ignore it at the root of repo,
+but it's nice to have, to make the init/new commands work well.
+
 ```
-$ adr init decisions/my-new-team
-  decisions/my-new-team/0001-record-architecture-decisions.md
+$ mkdir -p decisions/$team
+$ cd decisions/$team
+$ echo "." > .adr-dir
```

Comment:
If you want to be funny you can combine these three lines to a single one (in bash):

    install -D /dev/stdin decisions/$team <<< "."

(This is not a serious suggestion.)

Reply:
Is the "." special in some sense to the adr tool, or is this just a marker to get github to persist the directory, or other?

Reply:
I think it's a directory name, and the dot refers to the directory the file is in.

Reply:
yeah, it's tool related.  Lets the ADR cli tool operate on files in the same directory, which seems to be the best way to have a bunch of adrs in the same github repo.